File: javascript/raindrops/raindrops.js
export const convert = (x) => {
  let str = '';

  if (x % 3 === 0) str += 'Pling';
  if (x % 5 === 0) str += 'Plang';
  if (x % 7 === 0) str += 'Plong';

  return str || x.toString();
};

File: go/raindrops/raindrops.go
package raindrops

import ( "strconv" )

func Convert (num int) string {
  out := ""

  if num % 3 == 0 {
    out += "Pling"
  }

  if num % 5 == 0 {
    out += "Plang"
  }

  if num % 7 == 0 {
    out += "Plong"
  }

  if out == "" {
    out = strconv.Itoa(num)
  }

  return out

}
